how-to-video-processing.md

```diff
@@ -0,0 +1,22 @@
+# Access to raw video frame data
+
+Raw video frame data is available via the `VideoCapturer.OnVideoFrame` event.  The following code from the `PeerCC` sample application demonstrates its use.
+
+> Note: The queue below is bound to the `uiDispatcher` which means the callback will run on the UI thread.  As such it is possible to block the UI in the event handler.

```

Comment:
No, video frames are async and will not block the thread unlike audio -- but the GUI dispatcher is slower so it's possible in the setup to specify a specific thread for these dispatchers (video) too; and a different one from the audio handler.

Reply:
Fixed